x-pack/plugins/security_solution/server/endpoint/services/actions/action_list.test.ts

```diff
@@ -95,6 +95,7 @@ describe('When using `getActionList()', () => {
           agentType: 'endpoint',
           hosts: { 'agent-a': { name: 'Host-agent-a' } },
           command: 'kill-process',
+          alertIds: undefined,
```

Comment:
Why do we need these changes?

Reply:
The test needed to be updated as the API command list is now changed resulting in the endpoint action generator (that uses that) now picks the `execute` command for the tests instead.